examples/jdbc-atomic-batch-execute-operation/jdbc_atomic_batch_execute_operation.bal

```diff
@@ -35,45 +36,40 @@ public function main() returns error? {
     ];
 
     // Creates a batch parameterized query.
-    sql:ParameterizedQuery[] insertQueries = 
-        from var data in insertRecords
-        select `INSERT INTO Customers
+    sql:ParameterizedQuery[] insertQueries =
+        from var customer in customers
+    select `INSERT INTO Customers
```

Comment:
Shall we format this?

Reply:
+1

Reply:
Fixed in [3265b44](https://github.com/ballerina-platform/ballerina-distribution/pull/2842/commits/3265b44b42d74499757b3a169e228bacce987000)